default.yaml

```diff
@@ -125,6 +125,18 @@ rateLimits:
 
 accounts:
   initialStatus: active
+  # customFields:
+  #   - name: "customField"
+  #     type: <"string" | "integer" | "float" | "boolean">
+  #     defaultValue: <value> # optional
+  #     # default: false - true if field is an array of type
```

Comment:
this line is not clear to me. do you mean `default: false` expect when `type: array`, in that case default is true?

Reply:
ok I think i get it: `#default` refers to the line below. not the line above. I think that could be make more clear by adding some line breaks.